modules/exploits/windows/browser/ms12_004_midi.rb

```diff
@@ -10,19 +10,6 @@ class Metasploit3 < Msf::Exploit::Remote
 
   include Msf::Exploit::Remote::HttpServer::HTML
   include Msf::Exploit::RopDb
-  include Msf::Exploit::Remote::BrowserAutopwn
-  autopwn_info({
-    :ua_name    => HttpClients::IE,
-    :ua_minver  => "6.0",
```

Comment:
Yeah I am not really worried about it. MS12-043 and MS12-063 and seem better and they're always the ones popping shells anyway. MS12-004 for me fails more often so I decided to replace it.